### PR TITLE
Moved populate_instance_names

### PR DESCRIPTION
**Pull request by abrown-sg**
* Cleanup to move method into config class.
* Added tests.
* yapf cleaned up some unrelated test files

### CODE REVIEW

File: molecule/config.py [hunk 1]
```diff
@@ -113,3 +113,13 @@ def update_ansible_defaults(self):
         # grab config_file default from molecule if it's not set in the user-supplied ansible options
         if 'config_file' not in self.config['ansible']:
             self.config['ansible']['config_file'] = self.config['molecule']['config_file']
+
+    def populate_instance_names(self, platform):
+        """
+        Updates instances section of config with an additional key containing the full instance name
+
+        :return: None
```
Comment (rgreinho) [inline molecule/config.py]: Docstring for ``platform``.

**Comment by rgreinho** [comment]
:+1: aside my small remark.

**Comment by abrown-sg** [comment]
Ah, thanks, got that updated!

**Comment by retr0h** [comment]
:100:

**Pull request merged**